app/src/main/java/net/gsantner/markor/activity/DocumentEditAndViewFragment.java

```diff
@@ -219,6 +218,8 @@ public void onViewCreated(@NonNull View view, Bundle savedInstanceState) {
         }
         _webView.setBackgroundColor(Color.TRANSPARENT);
 
+        loadDocument();
+
```

Comment:
We wan to set the edit text undo redo helper _after_ the first load. Please reorganise this.